notebooks/loading-and-processing-data.ipynb

```diff
@@ -275,11 +275,11 @@
       "WARNING:esmvalcore.cmor.check:There were warnings in variable tas:\n",
       " Coordinate lon units degrees converted to degrees_east\n",
       " Coordinate lat units degrees converted to degrees_north\n",
-      "loaded from file /home/bandela/climate_data/cmip5/output1/CCCma/CanESM2/historical/mon/atmos/Amon/r1i1p1/v20120718/tas_Amon_CanESM2_historical_r1i1p1_185001-200512.nc\n",
+      "loaded from file ~/climate_data/cmip5/output1/MPI-M/MPI-ESM-MR/historical/mon/atmos/Amon/r1i1p1/v20120503/tas_Amon_MPI-ESM-MR_historical_r1i1p1_185001-200512.nc\n",
```

Comment:
These warnings do not appear whith this model, they're specific to CanESM2

Reply:
I re-ran the entire notebook after clearing all outputs, so I guess these warnings also appear for the other model. Nevertheless, I will wait until https://github.com/ESMValGroup/ESMValCore/pull/1945 is merged before further changes to the notebooks here.

Reply:
Strange, I don't see them when running the notebook for #1945.